app/controllers/v5/stories_controller.rb

```diff
@@ -0,0 +1,93 @@
+module V5
+  class StoriesController < V5::BaseUserController
+    power :stories, map: {
+      [:show] => :show_stories,
+      [:update] => :update_stories,
+      [:create] => :create_story
+    }, as: :story_scope
+
+    def index
+      load_stories
+      render_stories
+    end
+
+    def show
+      load_story or render_not_found
+      render_story
+    end
+
+    def create
+      if build_story
+        render_story :created
+      else
+        render_errors
+      end
+    end
+
+    def update
+      load_story or render_error('Invalid story id') && return
```

Comment:
`render_error('Invalid story id') and return unless load_story`

On Fri, Mar 18, 2016 at 3:12 PM, Brian Zoetewey <notifications@github.com>
wrote:

> In app/controllers/v5/stories_controller.rb
> <https://github.com/CruGlobal/measurements-api/pull/19#discussion_r56707910>
> :
>
> > +
> > +    def show
> > +      load_story or render_not_found
> > +      render_story
> > +    end
> > +
> > +    def create
> > +      if build_story
> > +        render_story :created
> > +      else
> > +        render_errors
> > +      end
> > +    end
> > +
> > +    def update
> > +      load_story or render_error('Invalid story id') && return
>
> load_story or render_error('Invalid story id') and return -- (fails)
> render_error('Invalid story id') and return unless load_story -- (passes)
> return render_error('Invalid story id') unless load_story -- (passes)
> Which looks/feels correct?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/CruGlobal/measurements-api/pull/19/files/883276eb04156e48148b4c8ebdaea29ccc5a4012#r56707910>
>